### DISCUSSION TITLE
Malware detected on 2.8.4 dmg file

### DISCUSSION THREAD

**Discussion by jqscreative** [DeFiCh/app#1119] [General]
Hi, my Antivirus software detected the MacOS:Miner-DZ[PUP] virus and gatekeeper also stopped the launch.

See screenshots.

I'm shocked this wasn't picked up in the update, as its a nasty Virus, however without anti virus software it can go undetected into macs even run before gatekeeper stops stops the launch.
![Screenshot 2022-01-13 at 01 59 02](https://user-images.githubusercontent.com/76595389/149257500-d13aa213-2d4d-4d1b-a408-f1d98afb880b.png)
![Screenshot 2022-01-13 at 02 09 44](https://user-images.githubusercontent.com/76595389/149257504-01497b44-d110-4dc2-a29a-27a9f2e349cc.png)

Can you guys please look into this.

Thx
Jay

**Comment by izzycsy** [comment]
Hi @jqscreative , thanks for opening this, moving it to [Issues](https://github.com/DeFiCh/app/issues/1122) for tracking and investigation.

**Comment by prasannavl** [comment]
Hi Jay - It's a false positive from AVG. Mining is an intended use and it's detecting it as an unintended use case (due to the way the malware detection works - usually looks for signatures of bitcoin mining code, which DeFiChain wallet does have). It's not uncommon for wallets to be flagged by Anti-virus software until they are whitelisted. Encourage the community to report to AVG as false positive so that they can whitelist it. Thanks.

Meanwhile, for added protection and verification, we do publish the SHA256 of the software in the releases page: <https://github.com/DeFiCh/app/releases> page. That can be used for anti-tampering and integrity verification.   

For further context: This can help https://support.avg.com/answers?id=9060N0000000Ro9QAE
